wordpress-mobile/MediaPicker-iOS
# Sort user albums alphabetically

This PR makes sure that user albums are sorted in alphabetic order, to make it easier to find them.

Keep in mind that this is only for user albums, smart albums are always show first in a pre-determined order.

To test:
 - Start the demo app
 - Open the picker pressing +
 - Check that user albums are sorted alphabetically after the smart albums.
